### DISCUSSION TITLE
Cleanup fails locally when calling AnonymousComponent::resolve()

### DISCUSSION THREAD

**Discussion by nickworks** [spatie/laravel-backup#1639] [Q&A]
Hello,

I'm using `spatie/laravel-backup` v8.1.4.
When I run `php artisan backup:run` everything works (locally and in production).
When I run `php artisan backup:clean` I get the following error (locally only):
```
Cleanup failed because: Call to undefined method Illuminate\View\AnonymousComponent::resolve() (View: /Users/nick/Projects/nha/nha-portal-api/vendor/laravel/framework/src/Illuminate/Notifications/resources/views/email.blade.php).
```
![Screen Shot 2023-02-01 at 9 35 36 AM](https://user-images.githubusercontent.com/439979/216077232-939e6c53-37d7-4d02-b0af-6f0e7cc388b0.png)

Any ideas what I'm doing wrong or how I can fix it? Thanks!

**Comment by freekmurze** [comment]
It seems to work here using the latest version of Laravel

**Comment by nickworks** [comment]
The following command fixes the issue for me: `composer update laravel/framework`
